docs/flashing.md

```diff
@@ -119,6 +119,36 @@ Flashing sequence:
 3. Flash a .hex file
 4. Reset the device into application mode (may be done automatically)
 
+## bootloadHID
+
+BootloadHID is a USB bootloader for AVR microcontrollers. The uploader tool requires no kernel level driver on Windows and can therefore be run without installing any DLLs.
+
+To ensure compatibility with the bootloadHID bootloader, make sure this block is present your `rules.mk`:
+
+    # Bootloader
```

Comment:
Might be a good idea to use the triple-backticks for this code block, for the makefile syntax highlighting.

Reply:
I would personally prefer to be sympathetic to the rest of the file, and potentially have a 2nd pass refactor using something like `remark-cli`.